react-commenting.js

```diff
@@ -7,19 +7,19 @@
 class App extends Component{
   constructor(){
     super()
-    // 1.
+    // 1. setting up the "variables"
     this.state = {
       userInput: "",
     }
   }
 
   robot = (e) => {
-    // 2.
+    // 2. creating a method to use HANDLEINPUT
     this.setState({ userInput: e.target.value})
   }
 
   render(){
-    // 3.
+    // 3. creating a function
```

Comment:
this one is destructuring state so you don't have to write `this.state.userInput` over and over again